xbmc/cores/paplayer/VideoPlayerCodec.cpp

```diff
@@ -298,12 +289,6 @@ void VideoPlayerCodec::DeInit()
     m_pDemuxer = NULL;
   }
 
-  if (m_pInputStream != NULL)
```

Comment:
Smart pointer are convenient but we should not blindly rely on them. Explicitly release the ref